index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome Books</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="./app.js" defer></script>
+</head>
+<body>
+  <h1>Awesome Books</h1>
+  <ul id="books">
+
+  </ul>
+  <form action="#">
+    <input type="text" placeholder="tilte" id="title"/>
+    <input type="text" placeholder="author" id="author"/>
+    <button type="button" id="add-btn">
+      Add book
+    </button>
+  </form>
+</html>
```

Comment:
- [ ] Please add the remove button in UI. It is a basic requirement.

Reply:
This issue is resolved. Nice job! :clap: :+1: